src/Controls/src/Build.Tasks/nuget/buildTransitive/net6.0-ios10.0/Microsoft.Maui.Controls.Build.Tasks.props

```diff
@@ -0,0 +1,9 @@
+<Project>
```

Comment:
We can do something, this was just the fastest. It should not matter the version here as the normal nuget fallback version will kick in. I was testing this with net7. However, we could rename to say `ios` and then use an include/link to include the `ios` folder as `$(_netversion)-ios` I'll have a look at that as it keeps the codebase cleaner...

Reply:
This is going to be a bit harder since there are 2 values that need to change: .NET version and OS verion. Not sure I can have a net12.0-ios10 if the net12 has a min version of the OS ios20...

Reply:
Going to leave this for now, and then in a new PR try and figure out some build logic for this. We need a version in the nuget package so it can install so this may mean seeing what the min supported version and things are.